app/components/shared/user-card.tsx

```diff
@@ -26,7 +26,9 @@ export function UserCard({ user }: Props) {
 
         <div className="flex flex-col justify-between">
           <div>
-            <CardTitle className="text-2xl">{user.name}</CardTitle>
+            <CardTitle className="text-2xl" data-testid="userCardName">
```

Comment:
```suggestion
            <CardTitle className="text-2xl">
```

Please avoid using `data-testid` as much as possible because it's invisible to the users (and screen readers).
Instead, you can rely on the fact that `CardTitle` renders an `h3` element, so you should be able to use `getByRole('heading', { name })` query.